.github/workflows/ci.yaml

```diff
@@ -97,15 +94,6 @@ jobs:
               with:
                   toolchain: ${{ matrix.rust_version }}
                   key: x-v3
-            - name: Pin dependencies to make it build with our MSRV
-              if: matrix.rust_version == '1.82'
-              shell: bash
-              run: |
-                 if [ ! -f ./Cargo.lock ]; then
-                   cargo update -p bytemuck_derive --precise 1.8.1
-                   cargo update -p typed-index-collections  --precise 3.2.3
-                   cargo update -p cfg-expr@0.20 --precise 0.17.0
-                 fi
```

Comment:
No longer needed since with the resolver="3", cargo should take the MSRV into account.